db/migrate/20230605100418_create_payments.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePayments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :payments do |t|
+      t.references :order, null: false, foreign_key: true
+      t.references :user, null: false, foreign_key: true
```

Comment:
I don't think user is required here. We can have user through order

Reply:
Done